### PR TITLE
Fix the Thanks section's displaying issues

### LINKED ISSUE #647: Thankyou for contribution not displaying names correctly
screenshot_15
I don't know what that thing is! First the name should be appearing on the top and not below and second the thing should be placed in the Contributors box and not outside it!
If anyone wants to work on this please tell :)

### PR DESCRIPTION
**Pull request by hoangvanthien**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request (replace the text as instructed)

- Fix #647 
- Besides, there WAS another problem not mentioned: contributors section didn't display all people. It somehow stopped fetching.
screen shot 2016-12-29 at 16 48 04
- I remove `contributors.js` because it is more convenient to fetch contributors list ONCE in `main.js`.

Comments: The simple solution for the #647 is adjusting `margin-top`. However, I found a really serious bug in `contributors.js` that variable `j` is not handled when it reaches the ending iterator of the array --> runtime error. Moreover, the JS code changing html content of `#name` **must not** be run in recursive function (i.e. while fetching the list). They are all fixed.

Live link: https://hoangvanthien.github.io

### CODE REVIEW

**Comment by gaeun** [comment]
LGTM, thanks!

**Comment by robbyoconnor** [comment]
@hoangvanthien there's a quirk you need to fix: 
image
It looks bad...

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Aside from that one quirk I mentioned, this is good...fix that and it's good to merge!

**Comment by hoangvanthien** [comment]
Erm... @robbyoconnor 
What browser and screen resolution are you using?
screen shot 2016-12-29 at 18 07 50

**Comment by codein-dwij** [comment]
http://prntscr.com/dp96jd 
Seems Fine for me :)

**Comment by robbyoconnor** [comment]
Google Chrome 56.0.2924.28

**Comment by robbyoconnor** [comment]
@codein-dwij -- it looks fine when the names come in...it's BEFORE the names are loaded that the issue arises. Look at my screenshot closely.

**Comment by robbyoconnor** [comment]
Oh Resolution is 1366x768

**Comment by robbyoconnor** [comment]
Still showing a problem: 

image

**Comment by robbyoconnor** [comment]
You have to account for the loading of the contributers to be delayed, so `everyone`(the filler text) shows.

**Comment by robbyoconnor** [comment]
Just double checking things -- sorry for noise.

**Comment by robbyoconnor** [comment]
bump!

**Comment by ankitrgadiya** [comment]
@hoangvanthien Please fix this. It's not looking good on small window.
screenshot from 2016-12-30 17-24-29
screenshot from 2016-12-30 17-25-53

**Comment by robbyoconnor** [comment]
@ankitrgadiya and if the contributors list isn't loaded -- `everyone` clobbers the `Thank you you for your contribution,` -- This whole feature is nice and all but it never really worked out all that well.

**Comment by robbyoconnor** [comment]
@hoangvanthien you working on this?

**Comment by hoangvanthien** [comment]
@robbyoconnor I am. Sorry for being quiet quite long. I had some other businesses that are done now.

**Comment by robbyoconnor** [comment]
@hoangvanthien CI failing is your fault: 
Here's what Coala said: 
```
|    | [MAJOR] BootLintBear:
|    | Since grid classes apply to devices with screen widths greater than or equal to the breakpoint sizes (unless overridden by grid classes targeting larger screens), `class="col-xs-12 col-sm-12 col-md-12 col-lg-12"` is redundant and can be simplified to `class="col-xs-12"`
```

File: index.html [hunk 1]
```diff
@@ -146,14 +146,14 @@
 
             <section id="contributors-thanks" href="#contributors">
                 <div id="contributors-thanks-list" class="row">
-                    <div class="col-md-12">
+                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
```
Comment (robbyoconnor) [inline index.html:149]: Just use `col-xs-12` -- it's redundant what you're doing.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

**Review by robbyoconnor — APPROVED** [review]

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Accidentally approved -- make the changes I mentioned and push it

**Comment by hoangvanthien** [comment]
I'm going to close this for experimenting. I will reopen it later.

File: index.html (hunk 1 above)
Comment (hoangvanthien) [reply index.html:149]: It is not as simple as that, I tried. Anyway, I fixed it here #759.

**Pull request closed without merging**